README.md

```diff
@@ -604,25 +606,25 @@ More information: https://developer.android.com/reference/android/app/Notificati
 
 Available options:
 
-"private" = NotficationCompat.VISIBILITY_PRIVATE
-"public" = NotficationCompat.VISIBILITY_PUBLIC
-"secret" = NotficationCompat.VISIBILITY_SECRET
+"private" = NotficationCompat.VISIBILITY_PRIVATE\
+"public" = NotficationCompat.VISIBILITY_PUBLIC\
+"secret" = NotficationCompat.VISIBILITY_SECRET 
 
 More information: https://developer.android.com/reference/android/app/Notification.html#VISIBILITY_PRIVATE
 
 ## Notification importance
 
-(optional) Specify `importance` to set importance of notification. Default value: "high"
+(optional) Specify `importance` to set importance of notification. Default value: IMPORTANCE.HIGH  
+Constants available on the `IMPORTANCE` object. `import PushNotification, {IMPORTANCE} from 'react-native-push-notification';`
 
 Available options:
 
-"default" = NotificationManager.IMPORTANCE_DEFAULT
-"max" = NotificationManager.IMPORTANCE_MAX
-"high" = NotificationManager.IMPORTANCE_HIGH
-"low" = NotificationManager.IMPORTANCE_LOW
-"min" = NotificationManager.IMPORTANCE_MIN
-"none" = NotificationManager.IMPORTANCE_NONE
-"unspecified" = NotificationManager.IMPORTANCE_UNSPECIFIED
+IMPORTANCE.DEFAULT = NotificationManager.IMPORTANCE_DEFAULT\
+IMPORTANCE.HIGH = NotificationManager.IMPORTANCE_HIGH\
+IMPORTANCE.LOW = NotificationManager.IMPORTANCE_LOW\
```

Comment:
There is some `\` at the end of some line, is there any reason ?

Reply:
Yeah the `\` adds a line break in markdown, making it easier to read.

Before:
![Screen Shot 2021-04-23 at 10 30 03 AM](https://user-images.githubusercontent.com/22333355/115908707-5ad85300-a41f-11eb-9070-33fab28356e7.png)


After:
![Screen Shot 2021-04-23 at 10 30 18 AM](https://user-images.githubusercontent.com/22333355/115908736-662b7e80-a41f-11eb-92ad-a3555ede2e86.png)